build/marionette.state.js

```diff
@@ -35,7 +39,7 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
     //   preventDestroy: {boolean} If true, then this will not destroy on `component` destroy.
     // }
     constructor: function constructor() {
-      var _ref = arguments[0] === undefined ? {} : arguments[0];
+      var _ref = arguments.length <= 0 || arguments[0] === undefined ? {} : arguments[0];
```

Comment:
Yes, definitely--I just want to introduce build artifacts in a version bump + build commit instead.  I'll take a look to see if that's included in the next update, which is getting ready to go out in the next day or two.